docs/plugins-dry.md

```diff
@@ -0,0 +1,106 @@
+---
+title: '"DRY" your document with plugins'
```

Comment:
What do you think about renaming this "Custom admonitions with a plugin"? That's a bit more tangible than the more abstract "DRY" (which I agree is important and maybe we can note this throughout?) and "how do I create a custom admonition type ?" Is one of the most common questions out there

Reply:
Where does "don't repeat yourself " fit in with the diataxis framework? It feels more abstract than a focused how-to. Maybe it would be best as a tutorial that shows a few different examples of DRY with plugins? I think either easy we should not key this bog us down so I'm fine just using the title you just suggested is you like